tests/thread/thread_stdin.py

```diff
@@ -0,0 +1,42 @@
+# test that having multiple threads block on stdin doesn't cause any issues.
```

Comment:
Upper case Test, since the sentence ends in a full stop.

Reply:
Happy to make this edit, but do want to point out the existing tests are *very* inconsistent in this regard. IIRC, the reason it's written like this is that I copied one of the other test files and tried to match its style.

Reply:
I agree there's a lot of inconsistency with comments.  But this jumped out at me, and I don't think any other tests have a lowercase start and a full stop (at least none of the thread tests).

Would be good to clean this up, but not really worth the time.